Source/WebCore/rendering/cocoa/RenderThemeCocoa.mm

```diff
@@ -2649,21 +2633,24 @@ static void adjustInputElementButtonStyleForVectorBasedControls(RenderStyle& sty
         style.setColor(buttonTextColor(styleColorOptions, !element->isDisabledFormControl()));
     }
 
+    // Enforce "line-height: normal".
+    style.setLineHeight(CSS::Keyword::Normal { });
```

Comment:
this also affects other menu list buttons right? like `<input type="date/time/month/week">`?